### PR TITLE
Feat: add Actor Consolidation database and repository

### PR DESCRIPTION
**Pull request by FirestarJes**
## References

* #0000

### CODE REVIEW

File: source/marketparticipant/Energinet.DataHub.MarketParticipant.ApplyDBMigrationsApp/Scripts/LocalDB/202410310950 Update existing certificate expirations.sql [hunk 1]
```diff
@@ -0,0 +1,3 @@
+UPDATE [dbo].[ActorCertificateCredentials]
```
Comment (defectiveAi) [inline source/marketparticipant/Energinet.DataHub.MarketParticipant.ApplyDBMigrationsApp/Scripts/LocalDB/202410310950 Update existing certificate expirations.sql:1]: ?

File: source/marketparticipant/Energinet.DataHub.MarketParticipant.Domain/Model/ActorConsolidation.cs [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+using System;
+
+namespace Energinet.DataHub.MarketParticipant.Domain.Model;
+
+public sealed class ActorConsolidation
+{
+    public ActorConsolidation(ActorId fromId, ActorId toId, DateTimeOffset scheduledAt, ActorConsolidationStatus status)
```
Comment (defectiveAi) [inline source/marketparticipant/Energinet.DataHub.MarketParticipant.Domain/Model/ActorConsolidation.cs:21]: No need to specify status when creating a new consolidation - status is always pending.

File: source/marketparticipant/Energinet.DataHub.MarketParticipant.Domain/Model/ActorConsolidation.cs [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+        ActorFromId = fromId;
+        ActorToId = toId;
+        ScheduledAt = scheduledAt;
+        Status = status;
+    }
+
+    public ActorConsolidationId Id { get; init; }
+    public ActorId ActorFromId { get; init; }
+    public ActorId ActorToId { get; init; }
+    public DateTimeOffset ScheduledAt { get; set; }
```
[the remark this answers is not in the document]
Comment (FirestarJes) [reply source/marketparticipant/Energinet.DataHub.MarketParticipant.Domain/Model/ActorConsolidation.cs:42]: i thought of that, but we aren't doing that anywhere else. But i'll change it

File: source/marketparticipant/Energinet.DataHub.MarketParticipant.Domain/Model/ActorConsolidation.cs (hunk 2 above)
Comment (FirestarJes) [reply source/marketparticipant/Energinet.DataHub.MarketParticipant.Domain/Model/ActorConsolidation.cs:21]: Fair point

**Review by defectiveAi — APPROVED** [review]

**Review by defectiveAi — APPROVED** [review]

**Pull request merged**